build.sbt

```diff
@@ -2,22 +2,23 @@ name := """editorial-permissions-client"""
 
 organization := "com.gu"
 
-licenses += ("MIT", url("http://opensource.org/licenses/Apache-2.0"))
+licenses += ("Apache-2.0", url("http://opensource.org/licenses/Apache-2.0"))
 
-scalaVersion := "2.10.4"
+scalaVersion := "2.11.7"
 
 crossScalaVersions := Seq("2.10.4", "2.11.4")
 
 scalacOptions ++= Seq("-feature", "-deprecation", "-language:higherKinds", "-Xfatal-warnings")
 
 libraryDependencies ++= Seq(
-  "com.amazonaws" % "aws-java-sdk" % "1.9.29",
-  "com.typesafe.akka" %% "akka-actor" % "2.3.4",
-  "com.typesafe.akka" %% "akka-agent" % "2.3.4",
-  "com.typesafe.akka" %% "akka-slf4j" % "2.3.4",
-  "net.liftweb" %% "lift-json" % "2.6",
-  "org.mockito" % "mockito-all" % "1.8.5" % "test",
-  "org.scalatest" %% "scalatest" % "2.2.4" % "test"
+  "com.amazonaws" % "aws-java-sdk-core" % "1.10.37",
```

Comment:
Thanks for this! It looks like [`aws-java-sdk-s3`](http://search.maven.org/#artifactdetails|com.amazonaws|aws-java-sdk-s3|1.10.37|jar) already depends on `aws-java-sdk-core`, so we could remove this explicit dependency here too.